### ISSUE TITLE
Site opens & closes before loading properly?

### ISSUE THREAD

**Issue by xcrowfangs** [ytdl-org/youtube-dl#22633]
## Checklist

- [x] I'm reporting a broken site support
- [x] I've verified that I'm running youtube-dl version **2019.09.28**
- [x] I've checked that all provided URLs are alive and playable in a browser
- [x] I've checked that all URLs and arguments with special characters are properly quoted or escaped
- [x] I've searched the bugtracker for similar issues including closed ones\

## Description

I've attached a video of what's happening when I try to open it- apologies if any details are unclear, I'm unsure of how to record my screen so it's just with my phone :/
Just a forewarning that I suck at coding but can usually make just enough sense of HTML etc. to pick pretty colours and all. Didn't install youtube-dl to do codes or anything, just to grab some music to upload to my iPod. (Yeah, I'm still stuck in 2006- but I don't have to listen to ads constantly, so I count that as a win lolol.) I also installed ffmpeg and I was using this tutorial: https://www.wikihow.com/Rip-Mp3s-from-Spotify
I've already checked that I'm running the newest version, uninstalled and reinstalled multiple times, changed the location of the files, restarted and updated my computer, turned Windows Defender off, used the troubleshoot function to run it on several different OS's, opened it as many different ways I could find, and basically anything else you could think of.
Basically- my issue is that I open it (doesn't matter if it's through Windows+R, open, run as administrator, or through the troubleshooter) and it does nothing until I fullscreen it, then it closes immediately, as should be evident in the video. This is the only computer I own so I can't test it on anything else. 
Video link here since I can't add it as an attachment: https://photos.app.goo.gl/aoFZQaYtwmeahxeS7

Thanks to anyone that can help.

**EDIT: I hope I submitted this in the right category. I literally only made a GitHub account for help with this lol

**Comment by dstftw** [comment]
Google: how to run command line app.
